app/Http/Requests/Company/StoreCompanyRequest.php

```diff
@@ -55,6 +56,8 @@ public function rules()
             }
```

Comment:
Safe to never check the expenes mailbox in the Store Request as this is configured afterwards.... so only need a check on update

Reply:
@turbo124 I guess I missed this one :(
Just removing the following line? 
$rules['expense_mailbox'] = new ValidExpenseMailbox($this->company->key, $this->company->account->isPaid() && $this->company->account->plan == 'enterprise');

Reply:
@turbo124 should this not be keep in place, to suppor this, when using api create company?